common/src/main/java/dev/isxander/yacl3/gui/controllers/ColorPickerElement.java

```diff
@@ -0,0 +1,435 @@
+package dev.isxander.yacl3.gui.controllers;
+
+import dev.isxander.yacl3.api.utils.Dimension;
+import dev.isxander.yacl3.api.utils.MutableDimension;
+import dev.isxander.yacl3.gui.YACLScreen;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiGraphics;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.util.Mth;
+
+import java.awt.*;
+
+public class ColorPickerElement extends ControllerPopupWidget<ColorController> {
```

Comment:
Food for thought: should this be called `ColorPickerWidget` to more clearly differentiate it from controller elements?

Reply:
I'm good with that. I'll wait and see what Xander thinks before refactoring it. 👍

Reply:
Yes!

Reply:
Sounds good! I'm pretty busy until the weekend(and maybe sometime into the weekend depending on how well I manage my time) but I will get to work on this as soon as I can. I also need to fix the `YaclScreen#clearPopupControllerWidget` method being called multiple times on one clear for the record(e.g. closing the color picker once calls it 2-3 times depending on how you close it).

Reply:
Should have everything changed now 👍